collector/kernel/main.cc

```diff
@@ -277,6 +278,12 @@ int main(int argc, char *argv[])
   cli::ArgsParser parser("Flowmill agent.");
   args::HelpFlag help(*parser, "help", "Display this help menu", {'h', "help"});
   args::ValueFlag<u64> filter_ns(*parser, "nanoseconds", "Gap between subsequent reports", {"filter-ns"}, 10 * 1000 * 1000ull);
+  auto socket_stats_interval_sec = parser.add_arg<double>(
+      "socket-stats-interval-sec",
+      "Interval between sending socket stats (floating point)",
```

Comment:
For the 'help' text, should we note this is in seconds?

Reply:
Yes, thought I did :^), will add.